src/inc-upgrade.php

```diff
@@ -833,6 +833,14 @@ function lovd_addConditionalSQL ($sCondition, $aConditionArgs, $sSQL)
                       VALUES ("' . $_CONF['refseq_build'] . '", "' . $_CONF['refseq_build'] . ' / ' .
                               $_SETT['human_builds'][$_CONF['refseq_build']]['ncbi_name'] . '", 0, NOW())',
                  ),
+                 '3.5-pre-03' => array(
+                     'ALTER TABLE ' . TABLE_VARIANTS .
+                     'ADD COLUMN source VARCHAR(4) ',
+                     'UPDATE ' . TABLE_VARIANTS . ' AS v' .
+                     ' SET source = (SELECT id FROM ' . TABLE_GENOME_BUILDS .
+                                    ' WHERE column_suffix = "")' .
+                     ' WHERE v.id NOT IN (SELECT id FROM ' . TABLE_VARIANTS_ON_TRANSCRIPTS . ')',
```

Comment:
Netjes!
Omdat `TABLE_VARIANTS` de enige tabel is in de `UPDATE`, is een alias niet nodig. Als je toch liever een alias wilt gebruiken, gebruik dan liever `vog` in plaats van `v`, om het verschil met de VOT tabel te verduidelijken.